Testing/functional/e2e/cypress/support/commands.js

```diff
@@ -8,7 +8,7 @@
 // https://on.cypress.io/custom-commands
 // ***********************************************
 const { AuthMethod } = require("./constants");
-
+let storedCookies = {};
```

Comment:
this looks odd. Is that because is defined at the bottom? If so, can the definition be moved before the 'login' command?

Reply:
yeah this is our common variable to store cookies to restore each as cypress clears cookies before each test (it). I don't think it is only for login though, we keep/store cookies after login as we need to restore them before each functional tests (if we need to).

Reply:
Right, but why is it defined as an empty object?

Reply:
ah good point, I will update it to an empty array then

Reply:
ah, would be a global variable as well? I guess what is odd to me is that this file contains commands that are being added to cypress. this variable also sets state

Reply:
@FuriousLlama any suggestions to move this global variable from commands.js to somewhere else? may be a new js file just for global variables?

Reply:
moving global var to a separate js file